Composer/packages/client/src/components/AddRemoteSkillModal/CreateSkillModal.tsx

```diff
@@ -53,14 +54,19 @@ export interface CreateSkillModalProps {
   onDismiss: () => void;
 }
 
-export const validateManifestUrl = async ({ formData, formDataErrors, setFormDataErrors }) => {
+export const validateManifestUrl = async ({ formData, formDataErrors, setFormDataErrors }, skills: string[] = []) => {
   const { manifestUrl } = formData;
   const { manifestUrl: _, ...errors } = formDataErrors;
 
   if (!manifestUrl) {
     setFormDataErrors({ ...errors, manifestUrl: formatMessage('Please input a manifest URL') });
   } else if (!urlRegex.test(manifestUrl)) {
     setFormDataErrors({ ...errors, manifestUrl: formatMessage('URL should start with http:// or https://') });
+  } else if (skills.includes(manifestUrl)) {
+    setFormDataErrors({
+      ...errors,
+      manifestUrl: formatMessage('The bot is already part of the Bot Project'),
```

Comment:
Can we change the message to something like “This skill has already been added to this project.” @vivekkshankar can you weigh in?

Reply:
This is proposed by Vivek. 
“This skill has already been added”

@alanlong9278 let's talk about the local scenario, can we do the same for adding local bot as a skill, do we need extra design here?

Reply:
> This is proposed by Vivek.
> “This skill has already been added”
> 
> @alanlong9278 let's talk about the local scenario, can we do the same for adding local bot as a skill, do we need extra design here?

Yes，It will pop the err dialog when adding duplicated local bot as skill now. So we need a new design for adding  duplicated local bot. @luhan2017 
![duplicated](https://user-images.githubusercontent.com/5860999/121659581-06735a80-cad5-11eb-831b-032c152dc22d.gif)
 duplicated local bot